app/controllers/collections_controller.rb

```diff
@@ -54,6 +54,16 @@ def new
     end
   end
 
+  def update
+    @collection = Collection.find(params[:id])
```

Comment:
```suggestion
```
We should check authentication and not just do a `find_by`. If you add `update` to the hook `load_and_authorize_resource`, `@collection` will become available and will be validated already.